public/app.js

```diff
@@ -8,7 +9,7 @@ $('.poll-submit-btn').on('click', function(e) {
 		const option4 = $('.option-input4').val();
 
 	$.ajax({
```

Comment:
This is the ajax call that I make when submitting the poll to create the URL. I console.log the id to make sure the ID is coming through like I think it should, and it seems to be.